packages/next/server-actions/client.ts

```diff
@@ -0,0 +1,105 @@
+'use server';
+
+import { createSession, type GetContextParameters, type InteractionMode } from '@logto/node';
+import NodeClient from '@logto/node/edge';
+
+import BaseClient from '../src/client';
+import type { LogtoNextConfig } from '../src/types.js';
+
+export type { LogtoContext, InteractionMode } from '@logto/node';
+
+export default class LogtoClient extends BaseClient {
+  constructor(config: LogtoNextConfig) {
+    super(config, {
+      NodeClient,
+    });
+  }
+
+  /**
+   * Init sign-in and return the url to redirect to Logto.
+   *
+   * @param cookie the raw cookie string
+   * @param redirectUri the uri (callbackUri) to redirect to after sign in
+   * @param interactionMode OIDC interaction mode
+   * @returns the url to redirect to and new cookie if any
+   */
+  async handleSignIn(
+    cookie: string,
+    redirectUri: string,
+    interactionMode?: InteractionMode
+  ): Promise<{ url: string; newCookie?: string }> {
+    const { nodeClient, session } = await this.createNodeClientFromHeaders(cookie);
+    await nodeClient.signIn(redirectUri, interactionMode);
```

Comment:
not sure if we need to create a client every time. can you take a look at https://github.com/logto-io/js/pull/636 and https://github.com/logto-io/js/blob/master/packages/sveltekit/src/index.ts#L102 to see if we can do the similar thing in next?